src/shared/lib/schemas/location.json

```diff
@@ -16,6 +16,7 @@
     "hospitalized": { "type": "number", "minimum": 0 },
     "discharged": { "type": "number", "minimum": 0 },
     "population": { "type": "number", "minimum": 0 },
+    "populationDensitySqMi": { "type": "number", "minimum": 0 },
```

Comment:
Let's use `populationDensity` and document it as square meters.

Reply:
Where should it be documented?

Reply:
@andys1376 I'll handle that in #505